scripts/app/gamAppAds.js

```diff
@@ -182,7 +184,11 @@ var ksfDfp = {};
             divToClose.style.display = "none";
             var headerToClose = document.getElementsByClassName('ksfDFPadHeader ' + slot);
             if (headerToClose[0] !== "undefined") {
```

Comment:
This will probably be always true. Should it be `headerToClose[0] !== undefined`?

Reply:
Good catch. I updated it to use typeof and also check for null which is the most likely problem in this case.